src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2563,6 +2575,11 @@ internal bool ClearHomeWorkspaceInternal()
 
                 model.ClearCurrentWorkspace();
 
+                var defaultWorkspace = Workspaces.FirstOrDefault();
+                //Every time that a new workspace is created we have to assign the Defautl Geometry Scaling value defined in Preferences (comming from DynamoSettings.xml)
```

Comment:
It seems we can remove `(comming from DynamoSettings.xml)`? If user, just modified the setting, then the setting is not serialized yet, as long as we mentioned `Defautl Geometry Scaling value defined in Preferences`, it should be fine

Reply:
Fixed comment in the next commit:
https://github.com/DynamoDS/Dynamo/pull/13656/commits/c20979418fa67d8906f184af3810952b71d26641